content/_changelogs/9.5.4.md

```diff
@@ -0,0 +1,65 @@
+## 9.5.4
+
+_Released 4/11/2022_
+
+**Bugfixes:**
+
+- Updates were made to silence Electron warnings related to being unable to
+  connect to dbus. These errors are normal and expected, and do not result in
+  test failures. Because they are always present when running Electron inside
+  docker containers, it has incorrectly led people to believe it is the
+  root-cause of an error within their test run. By silencing these errors, it
+  will improve the debug experience to allow users to focus on meaningful
+  warning and error messages. Fixed
+  [#19299](https://github.com/cypress-io/cypress/issues/19299).
+- Updates were made to ensure that all `*.enabled` events are sent if Cypress
+  becomes disconnected from the Chrome DevTools Protocol and must re-establish a
+  connection to ensure all command logs are displayed to the user. This fixes an
+  issue where some command logs, like download or network events, are missing
+  once Cypress has disconnected and then reconnected to the Chrome DevTools
+  Protocol. Fixed [#20618](https://github.com/cypress-io/cypress/issues/20618).
+- Fixed in issue where `cy.type('{enter}')` was not sending the Enter key for
+  Firefox `v98+`. This was not an issue with Firefox `v97` and below. Fixed
+  [#20562](https://github.com/cypress-io/cypress/issues/20562).
+- Fixed a regression in [9.3.0](/guides/references/changelog#9-3-0) where
+  updates made to split glob patterns provided to the `--spec` CLI parameter on
+  commas was incorrectly splitting the patterns. Fixes
+  [#20794](https://github.com/cypress-io/cypress/issues/20794).
+- Fixed an issue with `cy.root` to respect the
+  [`timeout`](/api/commands/root#Arguments) option passed to the command.
+  Previously, when the `timeout` option was provided, it was ignored and the
+  default timeout was used. Fixed
+  [#19985](https://github.com/cypress-io/cypress/issues/19985).
+- Updates were made to decrease the length of the Cypress cache path for Windows
+  to ensure installing pre-release versions of the Cypress binary are within the
+  maximum path length of 260 characters. This should improve the experience for
+  Windows users since it's been observed the path length has exceed the default
+  maximum, which resulted in install failures. Fixed in
+  [#20961](https://github.com/cypress-io/cypress/pulls/20961).
```

Comment:
I wouldn't mention this, it makes it sound like an improvement for all Windows users, this only affects pre-release installs. 

```suggestion
  maximum path length of 260 characters. Fixed in
  [#20961](https://github.com/cypress-io/cypress/pulls/20961).
```